etc/run_with_override.jl

```diff
@@ -40,8 +42,14 @@ tmpdepot = mktempdir(; cleanup=true)
 @info "Created temporary depot at $(tmpdepot)"
 
 # create override file for GAP_jll
-add_jll_override(tmpdepot, "GAP", gapoverride)
-add_jll_override(tmpdepot, "GAP_lib", gapoverride)
+add_jll_override(tmpdepot, "GAP", "5cd7a574-2c56-5be2-91dc-c8bc375b9ddf", gapoverride)
+add_jll_override(tmpdepot, "GAP_lib", "de1ad85e-c930-5cd4-919d-ccd3fcafd1a3", gapoverride)
+
+# trivially change JuliaInterface for rebuild
+run(`touch $(abspath(dirname(dirname(@__FILE__)), "pkg", "JuliaInterface", "src", "JuliaInterface.c"))`)
```

Comment:
that should be enough, shouldn't it?

Reply:
no, it computes a tree hash, you need to actually change the file content, or use something else as suggested in issue #997

Reply:
I like the idea of #997 and am already working on it.